src/version.h.in

```diff
@@ -26,4 +26,8 @@
 #define MIGRAPHX_VERSION_MINOR @PROJECT_VERSION_MINOR@
 #define MIGRAPHX_VERSION_PATCH @PROJECT_VERSION_PATCH@
 #define MIGRAPHX_VERSION_TWEAK "@PROJECT_VERSION_TWEAK@"
+#define MIGRAPHX_SO_MAJOR_VERSION           \
```

Comment:
```suggestion
#define MIGRAPHX_SO_VERSION_FLAT           \
```
Suggestion:
I wouldn't put `major` in the word.  MIopen uses `FLAT` to represent such version string.  Perhaps good idea to use same naming convention.

Reply:
CMakeLists.txt under AMDMIGraphX has this name, so I used that for consistency:
`math(EXPR MIGRAPHX_SO_MAJOR_VERSION "(${PROJECT_VERSION_MAJOR} * 1000 * 1000) + (${PROJECT_VERSION_MINOR} * 1000) + ${PROJECT_VERSION_PATCH}")`

I am open to changing the name if that is more desirable.

Reply:
Yea this is the major version for the SO.